File: Assignment__29/Question2.c
/*Accept number of rows and number of columns from user and display below pattern.
Input : iRow = 4 iCol = 4
Output : 2 4 6 8 10
         1 3 5 7 9
         2 4 6 8 10 
*/

#include<stdio.h>

void Display(int iRow, int iCol)
{
    int i = 0;
    int j = 0;

    for(i = 1; i <= iRow; i++)
    {
        int iNo1 = 2;
        int iNo2 = 1;

        for(j = 1; j <= iCol; j++)
        {
            if(i % 2 != 0)
            {
                printf("%d\t",iNo1);
                iNo1 = iNo1 + 2;
            }
            else
            {
                printf("%d\t",iNo2);
                iNo2 = iNo2 + 2;
            }
        }
        printf("\n");
    }
}


int main()
{
    int iValue1 = 0;
    int iValue2 = 0;

    printf("Enter the number of rows : \n");
    scanf("%d",&iValue1);

    printf("Enter the number of coloumns : \n");
    scanf("%d",&iValue2);

    Display(iValue1, iValue2);

    return 0;

}

File: Assignment__30/Question2.java
/*
Accept number of rows and number of columns from user and display below pattern. 
Input : iRow = 4 iCol = 4
Output : * * * *
         * * * #
         * * # #
         * # # # 
*/

import java.util.Scanner;

class Pattern
{
    public void Display(int iRow, int iCol)
    {
        int i = 0;
        int j = 0;

        if(iRow != iCol)
        {
            System.out.println("Invalid input.");
            System.out.println("Row number and coloumn number should be same.");

            return;
        }
        
        for(i = 1; i <= iRow; i++)
        {
            
            for(j = 1; j <= iCol; j++)
            {
              if((i >= 3 && j >= 3) )
              {
                System.out.print("#\t");
              }
              else
              {
                System.out.print("*\t");
              }
            }

            System.out.println();
        }
    }
}

public class Question2
{
    public static void main(String[] args)
    {
        Scanner sobj = new Scanner(System.in);
        int iValue1 = 0;
        int iValue2 = 0;

        System.out.print("Enter the number of Rows: \n");
        iValue1 = sobj.nextInt();

        System.out.print("Enter the number of Coloumns: \n");
        iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();
        pobj.Display(iValue1, iValue2);
    }
}

